react/package.json

```diff
@@ -24,11 +24,13 @@
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
     "typescript": "3.7.3",
-    "vtex.checkout-resources": "http://vtex.vteximg.com.br/_v/public/typings/v1/vtex.checkout-resources@0.11.0/public/_types/react",
-    "vtex.css-handles": "http://vtex.vteximg.com.br/_v/public/typings/v1/vtex.css-handles@0.4.1/public/@types/vtex.css-handles",
-    "vtex.order-items": "http://vtex.vteximg.com.br/_v/public/typings/v1/vtex.order-items@0.5.0/public/@types/vtex.order-items",
-    "vtex.order-manager": "http://vtex.vteximg.com.br/_v/public/typings/v1/vtex.order-manager@0.6.0/public/@types/vtex.order-manager",
-    "vtex.styleguide": "http://vtex.vteximg.com.br/_v/public/typings/v1/vtex.styleguide@9.94.0/public/_types/react"
+    "vtex.styleguide": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.styleguide@9.106.0/public/@types/vtex.styleguide",
```

Comment:
I think typings from the styleguide do not work properly, might get in your way during publish